serveradmin/servershell/static/js/servershell/command.js

```diff
@@ -235,6 +235,17 @@ servershell.commands = {
         // Now trigger reload with all changes at once
         servershell.shown_attributes = shown_attributes;
     },
+    bookmark: function(name) {
+        let attribute_ids = localStorage.getItem(name);
+
+        if (attribute_ids) {
+            servershell.shown_attributes = attribute_ids.split(',');
+        }
+        else {
```

Comment:
How do you overwrite a named bookmark?

Reply:
You don't: feature not implemented yet.

Reply:
I see ^^ .. might be interesting to leave a comment about that then here.

Reply:
Features have been added. There is now bookmark, bookmarkadd and bookmarkdel command.